src/index.html

```diff
@@ -8,6 +8,41 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      >
+
+      <div class="card__info">
+        <h4 class="card__info-text ">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h4>
+        <p class="card__text">Product code: 195434</p>
+
+        <div class="card__stars">
+          <ul class="card__stars-list">
+            <li><img src="images/star-active.svg" alt="gold--star"></li>
+            <li><img src="images/star-active.svg" alt="gold--star"></li>
+            <li><img src="images/star-active.svg" alt="gold--star"></li>
+            <li><img src="images/star-active.svg" alt="gold--star"></li>
+            <li><img src="images/star.svg" alt="void--star"></li>
```

Comment:
Consider using the layout from the previous task for the stars block to keep the modifier's functionality